README.md

```diff
@@ -331,4 +331,4 @@ Stats:
 ### New profiles:
 
 vmware-view, display-im6.q16, ipcalc, ipcalc-ng, ebook-convert, ebook-edit, ebook-meta, ebook-polish, lzop.
-avidemux, calligragemini, vmware-player, vmware-workstation, gget
\ No newline at end of file
+avidemux, calligragemini, vmware-player, vmware-workstation, gget, pipe-viewer
```

Comment:
(for the future) Please don't add new profiles to README.md/RELNOTES in PRs, it just create merge conflicts.